_dev/src/utils/TargetCountryValidator.ts

```diff
@@ -0,0 +1,14 @@
+import availableCountries from '../assets/json/countries.json';
+
+export function countryIsCompatibleWithGoogle(countryCode: string): string[] | [] {
```

Comment:
This name makes me think it returns a boolean. Would be better to call it `filter[...]`.

And we could accept a list of countries as input, and only return the compatible ones. This could fix in the future any merchant coming back with a incompatible country.